src/com/apeshko/javacore/task_01/Application.java

```diff
@@ -0,0 +1,244 @@
+                            .withType(Quadcopter.Type.MILITARY)
+                            .withHeight(100)
+                            .withWidth(100)
+                            .withLength(100.9)
+                            .withWeight(20)
+                            .withDistance(500)
+                            .build()
+                )
+            )
+    );
+
+    public static AirCompany getRandomAirCompany() {
+        return airCompanies.get(new Random().nextInt(airCompanies.size()));
+    }
+
+    public static void calculateCapacityTest() {
+        final AirCompanyService airCompanyService = new AirCompanyServiceImpl();
+        final AirCompany airCompany = getRandomAirCompany();
+
+        System.out.println("Capacity of the company " + airCompany.getName() + " equals " + airCompanyService.getCapacity(airCompany));
+    }
+
+    public static void calculateWeightTest() {
+        final AirCompanyService airCompanyService = new AirCompanyServiceImpl();
+        final AirCompany airCompany = getRandomAirCompany();
+
+        System.out.println("Carrying of the company " + airCompany.getName() + " equals " + airCompanyService.getWeight(airCompany));
+    }
+
+    public static void sortByDistanceTest() {
+        final AirCompanyService airCompanyService = new AirCompanyServiceImpl();
+        final AirCompany airCompany = getRandomAirCompany();
+
+        List<FlyingMachine> flyingMachines = airCompanyService.getMachinesSortByDistance(airCompany);
+
+        System.out.println("Flying machines of the company " + airCompany.getName() + " sorted by distance:");
+
+        flyingMachines
+                .stream()
+                .forEach(flyingMachine -> {
+                    System.out.println(" - " + flyingMachine.getName() + " " + flyingMachine.getDistance());
+                });
+    }
+
+    public static void searchByCriteriaTest() {
+        final AirCompanySearchService airCompanySearchService = new AirCompanySearchServiceImpl();
+        final AirCompany airCompany = getRandomAirCompany();
+        final FlyingMachineSearchCriteria flyingMachineSearchCriteria = new FlyingMachineSearchCriteria.Builder()
+                .withCapacityStart(50)
+                .withCapacityEnd(120)
+                .build();
+
+        List<FlyingMachine> flyingMachines = airCompanySearchService.search(airCompany, flyingMachineSearchCriteria);
+
+        System.out.println("Flying machines of the company " + airCompany.getName() + " with capacity in a range [" + flyingMachineSearchCriteria.getCapacityStart() + ", " + flyingMachineSearchCriteria.getCapacityEnd() + "]:");
+        flyingMachines
+                .stream()
+                .forEach(flyingMachine -> {
+                    System.out.println(" - " + flyingMachine.getName() + " " + flyingMachine.getCapacity());
+                });
+    }
+
+    static void makeFlightSuccessTest() {
+        final FlightServiceImpl flightService = new FlightServiceImpl();
```

Comment:
bad idea to create service instance within each method, better create service instance on class level, and access it everywhere you need (like interface it is a singleton)